numpy/core/src/multiarray/descriptor.c

```diff
@@ -238,6 +238,20 @@ is_datetime_typestr(char const *type, Py_ssize_t len)
     return 0;
 }
 
+// This function checks if val == {'name': any tuple}.
+// See gh-2865, issue #8235 for details
+inline npy_bool _check_if_size_1_and_contains_name_and_tuple(PyObject* val) {
```

Comment:
fix formatting and add `static`
```suggestion
static inline npy_bool 
_check_if_size_1_and_contains_name_and_tuple(PyObject* val) {
```

Reply:
Also change to `int`

Reply:
Also change to `int`